### DISCUSSION TITLE
[remote_config] add method to get json values

### DISCUSSION THREAD

**Discussion by mu-neeraj** [firebase/flutterfire#5605] [Feature request]
**Is your feature request related to a problem? Please describe.**
From firebase console you can set `json` value to a key. Flutter sdk does not expose any method to retrieve `json` type values. It only exposes methods to fetch primitive type values.   

**Describe the solution you'd like**
Flutter sdk to should expose a method 
``` dart
Map<String, dynamic> getJson(String key)
```

**Comment by Patryczon** [comment]
Same problem here :(

**Comment by bounty1342** [comment]
You can fetch the String value of the json and them json.decode() it. 

But there is an issue at the moment : https://github.com/FirebaseExtended/flutterfire/issues/2074

Please drop a like on it, so it gets prioritized 👍

**Comment by Ehesp** [comment]
So the native SDKs (web included) don't return a `getJson` method. I'll have a think about this one, however to actually check whether it's JSON we'd need to try catch the `getString` value when decoding it, which I'm not sure is ideal since it'd have to happen every time.
